core/src/main/scala/org/apache/spark/deploy/SparkCuratorUtil.scala

```diff
@@ -35,8 +35,11 @@ private[spark] object SparkCuratorUtil extends Logging {
   def newClient(
       conf: SparkConf,
       zkUrlConf: String = "spark.deploy.zookeeper.url"): CuratorFramework = {
```

Comment:
@tnachen did you miss this one? I don't think we need this param anymore since it's always going to be the same one.